src/sd_api_v5/sdk/components/serialization/application/codecs/common/conn_systemreset_app.c

```diff
@@ -0,0 +1,21 @@
+/* Copyright (c) 2014 Nordic Semiconductor. All Rights Reserved.
+ *
+ * The information contained herein is property of Nordic Semiconductor ASA.
+ * Terms and conditions of usage are described in detail in NORDIC
+ * SEMICONDUCTOR STANDARD SOFTWARE LICENSE AGREEMENT.
+ *
+ * Licensees are granted free, non-transferable use of the information. NO
+ * WARRANTY of ANY KIND is provided. This heading must NOT be removed from
+ * the file.
+ *
+ */
+
+#include "ble_serialization.h"
+#include "conn_systemreset.h"
```

Comment:
s/conn_systemreset.h/conn_systemreset_app.h/
The original conn_systemreset.h will not be needed, as we have replaced it's use with our declaration in sd_rpc.h

Reply:
Yes